image_view/src/image_view_node.cpp

```diff
@@ -214,7 +214,7 @@ void ImageViewNode::imageCb(const sensor_msgs::msg::Image::ConstSharedPtr & msg)
       }
     }
 
-    std::string encoding = msg->encoding.empty() ? "bgr8" : msg->encoding;
+    std::string encoding = "bgr8";
```

Comment:
this isn't the right fix - it would likely break mono8 usage.

If a particular encoding doesn't work for display, we should add a special rule for it below (see line 220-222 where we override bayer to mono8 - you can add another rule there for your yuv format that doesn't currently work)